lint_requirements.txt

```diff
@@ -3,7 +3,7 @@ black==25.1.0
 flake8==7.3.0
 flake8-pyproject==1.2.3
 isort==6.0.1
-mypy==1.18.1
+mypy==1.17.1
```

Comment:
Sounds like we should disable dependabot altogether. wdyt?

Reply:
I have no opinion. I never really found the head space to properly understand what we are using it for. I just assumed this keeps pulp-cli core and pulp-cli plugins consistent. If we do disable it, do we just never bump these versions again?

Reply:
First we need to template overwrite it https://github.com/pulp/pulp-cli/pull/1244 , then I believe you can delete that file. Or I add a post template hook to do that...